app/Controllers/Http/AdminsController.ts

```diff
@@ -0,0 +1,56 @@
+import type { HttpContextContract } from '@ioc:Adonis/Core/HttpContext'
+
+import Database from "@ioc:Adonis/Lucid/Database";
+import Profile from 'App/Models/Profile';
+
+export default class AdminsController {
+
+    public async getAllProfilewithPagination({ response, request }: HttpContextContract) {
+        const page = request.input('page', 1)
+        const limit = 10
+        const profile = await Database.from('profiles').paginate(page, limit)
```

Comment:
why use Database instance
lets follow active record pattern
use `Profile.query()...`

Reply:
btw, along with profile, wont you need `user` table details i.e email?